### DISCUSSION TITLE
支持画布缩放的动画有对应的API么？

### DISCUSSION THREAD

**Discussion by aquarius-wing** [antvis/X6#2022] [Q&A]
如果没有的话有什么好的实现思路？
我观察了下，调用graph.zoom*那几个接口似乎会修改好多dom的属性，单纯的加一个transition应该不行

**Comment by NewByVector** [comment]
没有的，你需要什么样的动画效果。

**Reply by aquarius-wing** [reply]
就比如是两帧起止画面分别是fitToContent和zoomToRect，然后中间衔接是其逐步方法聚焦到对应的rect。
就像是从整个宇宙逐步放大到银河系、太阳系、地球这样。
